framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -116,6 +116,9 @@
     // passing annotations to qualifierHierarchy.
     protected AnnotationMirror currentTop;
 
+    // Whether to work around Issue 979
+    protected final boolean shouldWorkAroundIssue979;
```

Comment:
I would name this something like `ignoreUninferredTypeArgs` so that we don't have to remember or look up what issue 979 is.